posthog/migrations/0420_set_all_survey_responses_to_be_strings.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.2.11 on 2024-06-04 15:17
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0419_remove_organization_available_features"),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql="""
+        update posthog_propertydefinition
+        set property_type = 'String', is_numerical=False
+        where name LIKE '$survey_response%' and property_type = 'Numeric'
```

Comment:
oh found a small problem here, we have some old survey properties on persons called `$survey_response/<surveyID>`  (instead of the new `$survey_responded/<surveyID>`). In this case, to be safer, we should only update event properties

```suggestion
        where name LIKE '$survey_response%' and property_type = 'Numeric' and type = 1
```

Reply:
(can you add this to the test case as well please 😅 )

Reply:
ooooh good shout!  Thank you!  I'll change this and add it to the tests :)